bedita-app/config/sql/upgrade/1-3.8-to-3.9-schema.sql

```diff
@@ -1 +1,3 @@
-ALTER TABLE `objects` ADD INDEX `objects_publisher_idx` (`publisher`);
\ No newline at end of file
+ALTER TABLE `objects` ADD INDEX `objects_publisher_idx` (`publisher`);
+
+INSERT INTO `object_types` (`id`, `name`, `module_name`) VALUES (42, 'caption', 'multimedia') ON DUPLICATE KEY UPDATE `id` = 42;
```

Comment:
in my local db I have `questionnaire_result` with `object_type_id` 42. Should we be worried about it? (42 already in use)